src/components/shared/header/header.jsx

```diff
@@ -1,11 +1,21 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 
-import Container from 'components/shared/container';
-
-const Header = () => (
-  <header>
-    <Container>Header!</Container>
+// TODO: Implement mobile menu functionality and delete eslint comment below, example — https://user-images.githubusercontent.com/20713191/144221747-70dc933e-a5bd-4586-9019-08117afc13e0.png
```

Comment:
I don't think having a screenshot with a snippet is a good option for illustrating examples. I'd prefer to have it in gist or a separate repository with snippets